### ISSUE TITLE
- operator on keypad

### ISSUE THREAD

**Issue by owenduffy** [qrp73/NanoVNA-Q#15]
The keypad does not seem to provide for entering a unary - operator.

A cancel key would also be useful.

![Screenshot-191111125312](https://user-images.githubusercontent.com/4297310/68555548-61207c80-0482-11ea-8507-4f1a901470c0.png)

Owen

**Comment by d51r3verse** [comment]
You seletected CH0/LOGMAG.
It always goes down below 0( If well calibrated ) = Not required to reference position set to below 0.
If you calibrate Through with attenuators, maybe minus reference point will helps for better resolution readings.

@qrp73 I asking to add PC function to send `trace` for auto_scale_Y and `marker` command for min/max( Cuz it seems to be not enough free code spaces )

**Comment by qrp73** [comment]
> A cancel key would also be useful.

you can cancel by pressing <= key (until it removes all digits and one time more)

**Comment by owenduffy** [comment]
Thanks, the <- key repeated closes the keypad without changing things so that is good... it was not one of the things I tried. I did try tapping outside the keypad (Android style) and that did not work.

What I have in mind is a little more flexibility of graph scaling. There is a scale factor and reference position, but one cannot specify the Y value at reference position.

Related, the phase graphs are pretty useless when they flick from +180 to -180 all the time. An option for 'phase continuous' where default setup is ref position=4, scale=45°/unit, and if a point would plot below the screen (ie phase<0°), add 360° so it plots on screen. This is a much more useful phase graph when phase hovers around 180°.

Owen

PS: the more I use -Q, the more I am convinced your fixup of the data entry by jog and keypad with a stylus is way ahead of the -H firmware... well done.
